identity.tf

```diff
@@ -0,0 +1,19 @@
+resource "azurerm_user_assigned_identity" "identity" {
```

Comment:
You need to associate this with the app gw https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/application_gateway#type-1

Reply:
Just added it,  running another plan on https://dev.azure.com/hmcts/PlatformOperations/_build/results?buildId=755465&view=results to see the results